src/spec/dsl.cr

```diff
@@ -64,6 +65,26 @@ module Spec
     exit
   end
 
+  # :nodoc:
+  class_getter randomizer_seed : Int32?
+  class_getter randomizer : Random::PCG32?
+
+  # :nodoc:
+  def self.order=(mode)
+    seed =
+      case mode
+      when "random"
+        Random::Secure.rand(1..99999) # 5 digits or less for simplicity
+      when /\A(\d{1,5})\Z/
```

Comment:
Please no Regex for parsing a number! Also I don't think there needs to be a limit enforced to the allowed seed number.